src/main/java/edu/wpi/first/shuffleboard/widget/BooleanBox.java

```diff
@@ -43,7 +43,7 @@ private Color getColor() {
     final Boolean data = getData();
     if (data == null) {
       return Color.BLACK;
-    } else if (data == true) { //NOPMD
```

Comment:
this was added specifically for readability. please revert it and disable the checkstyle check for this line.